packages/@vuepress/theme-default/src/client/components/ToggleDarkButton.vue

```diff
@@ -44,17 +44,39 @@ const toggleDark = (): void => {
   isDark.value = !isDark.value
 }
 
-const setDarkClass = (): void => {
+const updateDarkClass = (): void => {
   const htmlEl = window?.document.querySelector('html')
   htmlEl?.classList.toggle('dark', isDark.value)
 }
 
+const updateThemeColor = (): void => {
+  const themeColorEl = window?.document.querySelectorAll(
+    'meta[name="theme-color"]'
+  )
+  if (themeColorEl) {
```

Comment:
Let's return early to avoid nested `if`, e.g. `if (!xxxx) return`

Reply:
Thanks, fixed in commit https://github.com/vuepress/vuepress-next/pull/218/commits/644131a05d330d5588e23ea27a13f59c1452681d